prow/kube/helpers.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kube
+
+// LabelsMatch returns whether the two provided label sets match.
+// l1 needs to be a subset or match exactly l2.
+func LabelsMatch(l1, l2 map[string]string) bool {
+	for k1, v1 := range l1 {
+		matched := false
+		for k2, v2 := range l2 {
```

Comment:
This inner loop can be replaced with:
```
if v2, ok := l2[k1]; !ok || v1 != v2 {
   return false
}
```
And `return true` if the main for loop terminates.